src/aiidalab_qe/plugins/bands/setting.py

```diff
@@ -36,12 +36,23 @@ def __init__(self, **kwargs):
             disabled=False,
             style={"description_width": "initial"},
         )
-
+        self.two_dim_kpoints_path = ipw.Dropdown(
```

Comment:
```suggestion
        self.kpath_2d = ipw.Dropdown(
```
I guess this is better?